test/transaction/transaction.js

```diff
@@ -382,22 +382,36 @@ describe('Transaction', function() {
         return transaction.serialize();
       }).to.throw(errors.Transaction.FeeError.Different);
     });
+    it('checks output amount before fee errors', function() {
+      var transaction = new Transaction();
+      transaction.from(simpleUtxoWith1BTC);
+      transaction
+        .to(toAddress, 10000000000000)
+        .change(changeAddress)
+        .fee(5);
+
+      expect(function() {
+        return transaction.serialize();
+      }).to.throw(errors.Transaction.InvalidOutputAmountSum);
+    });
     describe('skipping checks', function() {
-      var buildSkipTest = function(builder, check) {
+      var buildSkipTest = function(builder, check, expectedError, opts) {
         return function() {
           var transaction = new Transaction();
           transaction.from(simpleUtxoWith1BTC);
           builder(transaction);
 
-          var options = {};
+          var options = opts || {};
           options[check] = true;
```

Comment:
Rather than check being an argument, couldn't we pass it more explicitly as part of the opts argument?